### DISCUSSION TITLE
recorder and egress

### DISCUSSION THREAD

**Discussion by huangshan309** [mynaparrot/plugNmeet-server#418] [General]
Just curious, is there any special reason not to use livekit's egress, but to implement another recorder?

**Comment by jibon57** [comment]
If you compare both then you'll easily understand.

**Comment by huangshan309** [comment]
In my opinion, the Room composite of livekit egress records a headless chrome, and plugNmeet-recorder implements similar functions, right? Although the methods are different, egress uses x11 and pulse audio, while plugNmeet-recorder uses mediarecorder, but the results are all the same, they both recording a chrome browser, so i think maybe we can use egress to implement the function of plugNmeet-recorder?

**Reply by jibon57** [reply]
We've whiteboard, shared note, chat ... Also egress is expensive because you can't run multiple session at same time in a single server.

**Reply by huangshan309** [reply]
thank you, i got it.

**Reply by smvelo** [reply]
Hi Iam getting this error when trying to login how can I resolve it, please kindly assist: ailed to load resource: net::ERR_CONNECTION_REFUSED
login.html:259 
        
        
       Uncaught (in promise) TypeError: Failed to fetch
    at sendRequest (login.html:259:32)
    at processRequest (login.html:218:27)
    at HTMLFormElement.<anonymous> (login.html:208:9)
